recipes-connectivity/remotemanager/remotemanager/remotemanager.sh

```diff
@@ -32,6 +163,10 @@ do
 			echo "sleeping for $IdleSleepTimeSec seconds"
 			sleep $IdleSleepTimeSec
 		fi
+		# Reloading configfile if was upated after the last connection
+		echo "----------------------------"
+    echo "Reloading Config File"
+		source $ConfigFile
```

Comment:
Check indentation of these two lines.

For safety put quotes around the variable: `source "$ConfigFile"`

Reply:
remove these lines